### ISSUE TITLE
Switch to using cargo-make

### ISSUE THREAD

**Issue by jayvdb** [juhaku/utoipa#1040]
Would you be interested in me replacing test.sh with a Makefile.toml

https://github.com/sagiegurari/cargo-make

I can do all the CI stuff to make this work.

**Comment by juhaku** [comment]
Why not, I have been actually weighting upon that or justfile or just plain make. :laughing: But I haven't got anywhere with that choice. Sure if you want to set things up that would be great. That test file get's anyway quite cumbersome to maintain. One thing to add is probably coverage as well at somepoint but haven't really got to it.

**Comment by jayvdb** [comment]
Both cargo-make and just are supported by https://github.com/taiki-e/install-action/blob/main/TOOLS.md , which speeds up the CI installing of the tool.

just has more stars - I havent used that yet, so I will try it first.

**Comment by juhaku** [comment]
Yeah, sure thing, for what I understand the justfile basically 60 % makefile and then the rest is build on top of it to enhance the quirks of old fashioned makefile to give smooth command running experience. 

For example the acitx-web is using justfile

**Comment by Drifter-J** [comment]
The proposed change is moot because a justfile was adopted. Hence closing.
